build/crunchy-postgres-exporter/Dockerfile

```diff
@@ -19,13 +20,22 @@ RUN if [ "$DFSET" = "centos" ] ; then \
     && ${PACKAGER} install -y \
         --setopt=skip_missing_names_on_install=False \
         postgresql${PGVERSION} \
+		gettext \
+		nss_wrapper \
     && ${PACKAGER} -y clean all ; \
 fi
 
+# needed for nss_wrapper
+RUN if [ "$BASEOS" = "rhel7" ] ; then \
+	${PACKAGER} -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm ; \
+fi
+
 RUN if [ "$DFSET" = "rhel" ] ; then \
 	${PACKAGER} install -y \
         --setopt=skip_missing_names_on_install=False \
         postgresql${PGVERSION} \
+		gettext \
+		nss_wrapper \
     && ${PACKAGER} -y clean all ; \
 fi
```

Comment:
Need to set this up for ubi7/ubi8.

Reply:
`DFSET` `rhel`  includes both `ubi7` and `ubi8`.

Reply:
OK.